platform/android/app/src/main/kotlin/coop/polypoly/polypod/OnboardingActivity.kt

```diff
@@ -73,9 +73,8 @@ class OnboardingActivity : AppCompatActivity() {
                 )
                 button.visibility = View.VISIBLE
                 button.setOnClickListener {
-                    Authentication.setUp(this) {
-                        Preferences.setBiometricEnabled(this, true)
-                        close()
+                    Authentication.setUp(this, newStatus = true) { _ ->
+                        true
```

Comment:
Is the callback needed at all?

Reply:
to cover both cases in Onboarding + MainFragment, it's not needed on the first but only on the second